packages/react-data-grid/src/Cell.tsx

```diff
@@ -1,35 +1,47 @@
 import React from 'react';
 import classNames from 'classnames';
+import shallowEqual from 'shallowequal';
 
 import { SubRowOptions, ColumnEventInfo, CellRenderer, CellRendererProps } from './common/types';
 import CellActions from './Cell/CellActions';
 import CellExpand from './Cell/CellExpander';
 import CellContent from './Cell/CellContent';
 import { isFrozen } from './ColumnUtils';
+import { isPositionStickySupported } from './utils';
 
-function getSubRowOptions<R>({ rowIdx, idx, rowData, expandableOptions: expandArgs }: Props<R>): SubRowOptions<R> {
+function getSubRowOptions<R>({ rowIdx, idx, rowData, expandableOptions: expandArgs }: CellProps<R>): SubRowOptions<R> {
   return { rowIdx, idx, rowData, expandArgs };
 }
 
-export interface Props<R> extends CellRendererProps<R> {
+export interface CellProps<R> extends CellRendererProps<R> {
   // TODO: Check if these props are required or not. These are most likely set by custom cell renderer
   className?: string;
   tooltip?: string | null;
   cellControls?: unknown;
 }
 
-export default class Cell<R> extends React.PureComponent<Props<R>> implements CellRenderer {
+export default class Cell<R> extends React.Component<CellProps<R>> implements CellRenderer {
   static defaultProps = {
     value: ''
   };
 
   private readonly cell = React.createRef<HTMLDivElement>();
 
+  shouldComponentUpdate(nextProps: CellProps<R>) {
```

Comment:
I noticed that cells are getting re-rendered on scroll because we [set](https://github.com/adazzle/react-data-grid/blob/next/packages/react-data-grid/src/Row.tsx#L61) `scrollleft` and it changes on scroll so `PureComponent` was just doing extra check without any benefit. `ScrollLeft` is needed only on [initial render](
https://github.com/adazzle/react-data-grid/blob/next/packages/react-data-grid/src/Cell.tsx#L34) when there are frozen columns to set the correct initial position. It is not needed after that so I am not checking it in SCU